pkg/agent/route/interfaces.go

```diff
@@ -38,6 +39,20 @@ type Interface interface {
 	// It should do nothing if the routes don't exist, without error.
 	DeleteRoutes(podCIDR *net.IPNet) error
 
+	// AddRoutes should add the route to the NodePort virtual IP.
+	AddNodePortRoute(isIPv6 bool) error
```

Comment:
What you think adding a separate interface for AddNodePortRoute, AddNodePort, and DeleteNodePort, which can have different implementations in Linux (by Route Client) and Windows (by Openflow)? Or maybe on Windows they are not needed at all, as the existing Openflow Client interfaces can already cover these functions?